cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/response/EmptySyncRequestResult.java

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright 2019 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
+ */
+package com.linkedin.kafka.cruisecontrol.servlet.response;
+
+import com.linkedin.kafka.cruisecontrol.servlet.parameters.CruiseControlParameters;
+
+public class EmptySyncRequestResult extends AbstractCruiseControlResponse {
```

Comment:
@alexwang08 Was this class redundant? Why did we remove it? To avoid redoing review for the same issue, can we either comment on a feedback to explain why it is not addressed or address the issue?

Reply:
The class was needed to pass in to `future.complete()` in my create sync request code in KafkaCruiseControlServlet before #569 got merged. Afterwards it was outdated and thus removed.